packages/@webex/plugin-meetings/src/member/index.ts

```diff
@@ -29,6 +29,7 @@ export default class Member {
   isRecording: any;
   isRemovable: any;
   isSelf: any;
+  isAssociatedWithSelf: boolean; // true for a device that we are paired with
```

Comment:
I am not sure about this one

Reply:
I could name it `isDevicePairedWithSelf` to make it more clear, but in theory this could be set to true for any type of participant not just a device, I don't know if that's possible in webex to pair with something else than a device or if maybe in the future we will have such feature, but in theory, based on just what Locus sends us, it is possible, so that's why I kept the name more generic - it matches what Locus sends us - they just have a field "associatedWith" that points to another participant and it could be anything not only a device.

Or we could rename it to "isPairedWithSelf"?